docs/mui-vale/styles/MUI/MuiBrandName.yml

```diff
@@ -1,22 +0,0 @@
-# Without a non-breaking space, brand names can be split in the middle
-# with the start and end on two different lines.
-# For example, Apple does this meticulously with their brand name: https://www.apple.com/macbook-air/.
-# Also read https://www.chrisdpeters.com/blog/using-non-breaking-spaces-to-help-with-branding/ for why.
-extends: substitution
-message: Use a non-breaking space (option+space on Mac, Alt+0160 on Windows or AltGr+Space on Linux, instead of space) for brand name ('%s' instead of '%s')
-level: error
-ignorecase: true
```

Comment:
I just realized that https://www.npmjs.com/package/markdownlint-rule-search-replace is case-sensitive. I tried using regexp in `serachPattern`, but it's terribly slow.

Maybe moving it to `markdowlint` wasn't a great idea 😅

@samuelsycamore Do you want to give [vale VS Code extension](https://marketplace.visualstudio.com/items?itemName=ChrisChinchilla.vale-vscode) a try? You will also need to [install Vale](https://vale.sh/docs/vale-cli/installation/)